app/db/models/user.js

```diff
@@ -228,13 +243,16 @@ module.exports = (Sequelize, db) => {
       firstName: this.getDataValue('firstName'),
       lastName: this.getDataValue('lastName'),
       profilePicture: this.getDataValue('profilePicture'),
+      graduationYear: this.getDataValue('graduationYear'),
+      major: this.getDataValue('major'),
```

Comment:
I think this and the other route might be caused by merge conflicts since `master`'s been pushed a bunch since this branch was originally created.

Reply:
Merged master and it worked fine